megamek/src/megamek/common/weapons/ArtilleryBayWeaponIndirectFireHandler.java

```diff
@@ -181,18 +181,34 @@ public boolean accept(Entity entity) {
             // Out of any valid spotters, pick the best.
             while (spottersAfter.hasNext()) {
                 Entity ent = spottersAfter.next();
-                if ((bestSpotter == null)
-                        || (ent.getCrew().getGunnery() < bestSpotter.getCrew()
-                                .getGunnery())) {
+                if (bestSpotter == null) {
                     bestSpotter = ent;
+                } else {
```

Comment:
This addition doesn't make much sense to me.  What is the purpose of the continue line? Is that meant to be a break?  It seems like the the first two conditions should be an if/else if.  This code is repeated again later on in ArtilleryWeaponIndirectFireHandler.